server/.directory

```diff
@@ -0,0 +1,6 @@
+[Dolphin]
+Timestamp=2020,4,6,11,39,35
+Version=4
+
+[Settings]
+HiddenFilesShown=true
```

Comment:
Is this something that should be in .gitignore?

Reply:
yep, removed